test/solvers/test_tree_dmrg.jl

```diff
@@ -0,0 +1,64 @@
+using Test: @test, @testset
+using ITensors
+using TensorOperations # Needed to use contraction order finding
```

Comment:
```suggestion
using TensorOperations: TensorOperations # Needed to use contraction order finding
```
That only brings `TensorOperations` itself into the namespace.